wurst/objects/abilities/Omnicure.wurst

```diff
@@ -40,19 +40,24 @@ class Omnicure extends ChannelAbilityPreset
         this.setArtDuration(1, 0)
 
 
+public class OmnicureItem extends Omnicure
+    construct(int newAbilityId)
+        super(newAbilityId, "", new Pair(0, 0), 0)
+        this.setButtonPositionNormalX(0)
+        this.setButtonPositionNormalY(0)
+
 @compiletime function createOmnicure()
-    new Omnicure(ABILITY_OMNICURE, "E", new Pair(1, 0))
+    new Omnicure(ABILITY_OMNICURE, "E", new Pair(1, 0), MANACOST)
+    new Omnicure(ABILITY_OMNICURE_ITEM, "E", new Pair(1, 0), 0)
+
+public function castOmnicure()
+    let heroes = GetUnitsInRectMatching(bj_mapInitialPlayableArea, Condition(() -> begin
```

Comment:
Can you not do this in a less archaic way? Using filters, condition, and code in Wurst seems silly. `ClosuresForGroups` should replace this.

Reply:
Updated

Reply:
Can you use the `nullTimer` for the dummy cast, similar to how cure all was fixed.